subprojects/packagefiles/theora/lib/meson.build

```diff
@@ -0,0 +1,205 @@
+encoder_sources = files(
```

Comment:
muon's fmt automatically sorts files like this. Probably should do the same.

Reply:
Took me a bit, but it's done.